SiReCore/utterance.py

```diff
@@ -84,12 +86,13 @@ def __init__(self, lab, args):
       self.txtloaded = True
     else:
       raise SiReError("Don't know what to do with intype - {0}".format(args.intype))
-    
+
     #Construct the utt from the proto utt
     self.id = proto["id"]
     self.phonemes = []
     self.syllables = []
     self.words = []
+    self.upper_reg = re.compile(r'[A-Z][A-Z]+')
```

Comment:
In Python there is a built in method for checking if a string is upper case:
str.isupper()
Please use that (returns true if all of hte string is upper case and false if not)

Reply:
I.e. the regular expression is not necessary